plugins/woocommerce/templates/order/source-data-fields.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Display the Source Data metabox.
+ *
+ * This template is used to display the order source data metabox on the edit order screen.
+ *
+ * @see     Automattic\WooCommerce\Internal\Orders\SourceAttributionController
+ * @package WooCommerce\Templates
+ * @version 8.4.0
+ */
+
+declare( strict_types=1 );
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Variables used in this file.
+ *
+ * @var array $meta Array of meta data.
+ */
+?>
+
+<div class="order-source-attribution-metabox">
+
+	<?php if ( array_key_exists( 'origin', $meta ) ) : ?>
+		<h4><?php esc_html_e( 'Origin', 'woocommerce' ); ?></h4>
+		<span class="order-source-attribution-origin">
+			<?php echo esc_html( $meta['origin'] ); ?>
+		</span>
+	<?php endif; ?>
+
+	<div class="woocommerce-order-source-attribution-details">
+
+		<a href="" class="woocommerce-order-source-attribution-details-toggle" aria-expanded="false">
+			<?php esc_html_e( 'Details', 'woocommerce' ); ?>
+			<span class="toggle-indicator" aria-hidden="true"></span>
+		</a>
+
+		<div class="clear"></div>
+
+		<div class="woocommerce-order-source-attribution-details-container closed">
+			<?php if ( array_key_exists( 'type', $meta ) ) : ?>
+				<h4><?php esc_html_e( 'Source type', 'woocommerce' ); ?></h4>
+				<span class="order-source-attribution-source_type">
+					<?php echo esc_html( $meta['type'] ); ?>
+				</span>
+			<?php endif; ?>
+
+			<?php if ( array_key_exists( 'utm_campaign', $meta ) ) : ?>
+				<h4><?php esc_html_e( 'UTM campaign', 'woocommerce' ); ?></h4>
+				<span class="order-source-attribution-utm-campaign">
+					<?php echo esc_html( $meta['utm_campaign'] ); ?>
+				</span>
+			<?php endif; ?>
+
+			<?php if ( array_key_exists( 'utm_source', $meta ) ) : ?>
+				<h4><?php esc_html_e( 'UTM source', 'woocommerce' ); ?></h4>
+				<span class="order-source-attribution-utm-source">
+					<?php echo esc_html( $meta['utm_source'] ); ?>
+				</span>
+			<?php endif; ?>
+
+			<?php if ( array_key_exists( 'utm_medium', $meta ) ) : ?>
+				<h4><?php esc_html_e( 'UTM medium', 'woocommerce' ); ?></h4>
+				<span class="order-source-attribution-utm-medium">
+					<?php echo esc_html( $meta['utm_medium'] ); ?>
+				</span>
+			<?php endif; ?>
+
+		</div>
+	</div>
+
+	<?php if ( array_key_exists( 'device_type', $meta ) ) : ?>
+		<h4><?php esc_html_e( 'Device type', 'woocommerce' ); ?></h4>
+		<span class="order-source-attribution-device_type">
+			<?php echo esc_html( $meta['device_type'] ); ?>
+		</span>
+	<?php endif; ?>
+
+	<?php if ( array_key_exists( 'session_pages', $meta ) ) : ?>
+		<h4>
+			<?php
+			esc_html_e( 'Session page views', 'woocommerce' );
+			echo wc_help_tip(
+				__(
+					'The number of unique pages viewed by the customer prior to this order.',
+					'woocommerce'
+				)
+			); // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
```

Comment:
As mentioned elsewhere, `wc_help_tip` runs a filter after sanitization that might change what it returns, I'd prefer we escape the output here.

Reply:
✔️ b9542765c1ecd84b5183608e09035e76b9f7fd77